lib/util.js

```diff
@@ -564,12 +586,12 @@ function formatValue(ctx, value, recurseTimes) {
     braces[0] = `${constructor.name} ${braces[0]}`;
 
   if (empty === true) {
-    return braces[0] + base + braces[1];
+    return `${braces[0]}${base}${braces[1]}`;
```

Comment:
I'm not sure it improves readability in this specific case, but LGTM either way.